guides/publishing-new-versions.md

```diff
@@ -0,0 +1,16 @@
+# Publishing New Versions
+
+The process to publish a new package version has attempted to find the middle ground between a hands off approach, and automation. Tending to not lean to heavily into automation.
```

Comment:
```suggestion
The process to publish a new package version is one that tries to find the middle ground between a hands on approach, and automation. Tending to not lean to heavily into automation.
```

I assume that's what you meant? since "hands off" and "automation" are kind of the same thing I believe

Reply:
Lol, yes this is what I meant. Thank you